data/json/items/armor/boots.json

```
@@ -201,7 +201,7 @@
     "type": "ARMOR",
     "category": "armor",
     "name": { "str": "pair of survivor fireboots", "str_pl": "pairs of survivor fireboots" },
-    "description": "A pair of customized, Kevlar armored Nomex boots, modified to provide maximum protection from harm and the elements, even when knee-deep in the dead.",
+    "description": "A pair of customized Kevlar armored Nomex boots.  These will keep you alive, even when knee-deep in the flames.",
```

```suggestion
    "description": "A pair of customized, Kevlar-armored Nomex boots.  These will keep you alive, even when knee-deep in the flames.",
```
"alive" might be a bit too strong a word, here and below; protected, maybe?